sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -133,6 +133,20 @@ builder.Services.AddOpenTelemetry().UseAzureMonitor(o =>
 });
 ```
 
+#### Disable Logs Sampling
+
+By default, logs emitted within an Activity context are sampled based on the configured [sampling ratio](#customizing-sampling-percentage) for traces. This sampling can be disabled as shown below.
+
+``` C#
+builder.Services.AddOpenTelemetry().UseAzureMonitor(o =>
+{
+    o.DisableTraceBasedSamplingForLogs = true;
```

Comment:
I have a general question regarding how to organize the Boolean options, do we want a mixure of "EnableXyz" and "DisableXyz", or it'll be better to stick with "EnableXyz"?

For example, "EnableTraceBasedSamplingForLogs = false".

Reply:
For this particular name, it is driven by the specification consistent with all languages.

But in general, the choice is based on

1) Name should reflect the intent clearly.
2) Consistency with other options when possible.

Disable makes the intent more clear than enable with default `true`. Right now, we are deviating on `EnableLiveMetrics = true` which I think should be changed to Disable as well.